static/js/components/AnonymousMenu_test.js

```diff
@@ -0,0 +1,39 @@
+// @flow
+import React from "react"
+import { assert } from "chai"
+import { shallow } from "enzyme"
+
+import AnonymousMenu from "./AnonymousMenu"
+import { routes } from "../lib/urls"
+
+describe("AnonymousMenu component", () => {
+  it("has a link to login", () => {
+    assert.equal(
+      shallow(<AnonymousMenu useScreenOverlay={false} />)
+        .find("MixedLink")
+        .at(0)
+        .prop("dest"),
+      routes.login
+    )
+  })
+
+  it("has a link to register", () => {
+    assert.equal(
+      shallow(<AnonymousMenu useScreenOverlay={false} />)
+        .find("MixedLink")
+        .at(1)
+        .prop("dest"),
+      routes.register.begin
+    )
+  })
+
+  it("has a link to create account", () => {
+    assert.equal(
+      shallow(<AnonymousMenu useScreenOverlay={false} />)
+        .find("MixedLink")
+        .at(1)
+        .prop("dest"),
+      routes.register.begin
+    )
+  })
```

Comment:
This is an exact copy of the test case above. Did you mean to test with `useScreenOverlay={true}` here?

Reply:
It doesn't matter either we go with the desktop or mobile version. In both cases, we will get that link for Create Account.